include/cloudbreak.bash

```diff
@@ -158,6 +169,20 @@ gen-password() {
     date +%s | checksum sha1 | head -c 10
 }
 
+generate-cert() {
```

Comment:
in order to avoid function name collision across different bash files every function is prefixed with the file name. e.g. it shall be cloudbreak-generate-cert